src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,71 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    public static final String DELIMITER = ",";
+    public static final String OPERATION = "buy";
+    private int supplyAmount = 0;
+    private int buyAmount = 0;
+    private final StringBuilder stringBuilder = new StringBuilder();
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
The method `getStatistic` should be divided into smaller methods to adhere to the single responsibility principle. For example, a separate method for reading the file data, processing it, and another for writing the result to the file.